3rdparty/dxc/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 list(APPEND NBL_DXC_CMAKE_OPTIONS "-DHLSL_OPTIONAL_PROJS_IN_DEFAULT:BOOL=OFF")
-list(APPEND NBL_DXC_CMAKE_OPTIONS "-DHLSL_ENABLE_ANALYZE:BOOL=OFF")
+# list(APPEND NBL_DXC_CMAKE_OPTIONS "-DHLSL_ENABLE_ANALYZE:BOOL=OFF")
```

Comment:
this file has been hacked away to get this to compile I think i can undo quite a bit of this.

Reply:
@AnastaZIuk can you look into his changes?